src/ubuntu/manifest.json

```diff
@@ -556,6 +556,22 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/ubuntu/20.04/webassembly",
```

Comment:
Do we need a 20.04 version for helix/wasm too? [18.04/helix/wasm](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/18.04/helix/webassembly/Dockerfile)

Reply:
I'm not sure I understand all of the context. Let's do it in another PR.

Perhaps just adding wasmtime to `src/ubuntu/20.04/helix/wasm/amd64/Dockerfile` would do ?